components/Cards/cards.css

```diff
@@ -0,0 +1,116 @@
+/* global css sheet for fonts and icons */
+@import "/css/main.css";
+
+/* main card component */
+.card {
+    height: 14rem;
+    width: 40rem;
+    border: 2px solid var(--universal-border-color);
```

Comment:
Don't use fixed width and height for the card as the card content is not in your hands.
So replace height by min-height
And width by max-width or min-width whichever suits best to you